jetson/percep/main.cpp

```diff
@@ -121,14 +121,13 @@ int main() {
     #if OBSTACLE_DETECTION && !WRITE_CURR_FRAME_TO_DISK
     
     #if PERCEPTION_DEBUG
-    //Update Original 3D Viewer
-    viewer_original->updatePointCloud(pointcloud.pt_cloud_ptr);
-    viewer_original->spinOnce(10);
-    cerr<<"Original W: " <<pointcloud.pt_cloud_ptr->width<<" Original H: "<<pointcloud.pt_cloud_ptr->height<<endl;
+      //Update Original 3D Viewer
+      pointcloud.updateViewer(originalView);
```

Comment:
4 spaces in a tab. I think VS Code auto-detects the number of spaces in a tab, so since throughout main it seems like we do 2 spaces for tab a lot when you hit tab on the keyboard it is indenting two spaces, while if you went to pcl.cpp and hit tab your tab would be 4 spaces, since all tabs in that file are 4 spaces

Reply:
Fixed